backend/inspirehep/indexer/tasks.py

```diff
@@ -109,6 +109,13 @@ def index_record(self, uuid, record_version=None, force_delete=None):
                 error=err,
             )
 
+    uuids_to_reindex = get_references_to_update(record)
+
+    if uuids_to_reindex:
+        batch_index(list(uuids_to_reindex))
+
+
+def get_references_to_update(record):
```

Comment:
since is used also in other modules, it should be moved as a method to `LiteratureRecord`. Isn't used only for literature?

Reply:
No, it's also for Authors and Conferences (see line 129 and 134). If you prefer we can do the record method for those record classes.

Reply:
Shall we move it to `indexer/api.py` then? it shouldn't be part of the tasks :)